includes/admin/discounts/contextual-help.php

```diff
@@ -25,12 +25,9 @@ function edd_discounts_contextual_help() {
 		'<p><strong>' . sprintf( __( 'For more information:', 'easy-digital-downloads' ) . '</strong></p>' .
 		'<p>' . sprintf( __( 'Visit the <a href="%s">documentation</a> on the Easy Digital Downloads website.', 'easy-digital-downloads' ), esc_url( 'http://docs.easydigitaldownloads.com/' ) ) ) . '</p>' .
 		'<p>' . sprintf(
-					__( '<a href="%s">Post an issue</a> on <a href="%s">GitHub</a>. View <a href="%s">extensions</a> or <a href="%s">themes</a>.', 'easy-digital-downloads' ),
-					esc_url( 'https://github.com/easydigitaldownloads/Easy-Digital-Downloads/issues' ),
-					esc_url( 'https://github.com/easydigitaldownloads/Easy-Digital-Downloads' ),
-					esc_url( 'https://easydigitaldownloads.com/downloads/?utm_source=plugin-discounts-page&utm_medium=contextual-help-sidebar&utm_term=extensions&utm_campaign=ContextualHelp' ),
-					esc_url( 'https://easydigitaldownloads.com/themes/?utm_source=plugin-discounts-page&utm_medium=contextual-help-sidebar&utm_term=themes&utm_campaign=ContextualHelp' )
-				) . '</p>'
+			__( 'Need more from your Easy Digital Downloads store? <a href="%s">Upgrade Now</a>!', 'easy-digital-downloads' ),
+			esc_url( 'https://easydigitaldownloads.com/pricnig/?utm_source=plugin-settings-page&utm_medium=contextual-help-sidebar&utm_term=pricing&utm_campaign=ContextualHelp' ),
```

Comment:
```suggestion
			esc_url( 'https://easydigitaldownloads.com/pricing/?utm_source=plugin-settings-page&utm_medium=contextual-help-sidebar&utm_term=pricing&utm_campaign=ContextualHelp' ),
```

Reply:
Actually I see this change in multiple (5) places. What would you think of creating a helper function...maybe make a new `functions.php` file in `admin/promos` or some place, and have it return the entire `sprintf()` string? Then the typo would be in just one place to fix, and you could add an optional parameter to swap out the `utm_source` if you wanted to (although they all look to be the same now).